client/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "cookies-next": "^4.2.1",
+    "font-awesome": "^4.7.0",
```

Comment:
Don't use external library, instead use custom icons, see icons folder how we are creating it and using it.